force-app/main/default/classes/AFFL_MultiRecordTypeMapper.cls

```diff
@@ -333,29 +316,30 @@ public virtual with sharing class AFFL_MultiRecordTypeMapper {
      * @param affl The affiliation to check for program enrollment creations
```

Comment:
We may need to fix the comment here since we have refactored the parameter.

Reply:
Updated.